src/test/utils.tsx

```diff
@@ -3,30 +3,12 @@ import { render as rtlRender } from "@testing-library/react";
 import { Provider as IntlProvider } from "../locale";
 import { Provider as ThemeProvider } from "../theme";
 import { MockedProvider as MockedGqlProvider, MockedResponse } from "@apollo/client/testing";
-import { LANGUAGE } from "@reearth/locale/queries";
 
-const queryMocks: readonly MockedResponse<Record<string, any>>[] | undefined = [
-  {
-    request: {
-      query: LANGUAGE,
-      variables: {},
-    },
-    result: {
-      data: {
-        me: {
-          id: "whatever",
-          name: "mock",
-          email: "mock@example.com",
-          lang: "en",
-          myTeam: { id: "whatever", name: "sample", __typename: "Team" },
-          auths: ["auth0"],
-          __typename: "User",
-        },
-      },
-    },
-  },
-];
-const render = (ui: React.ReactElement, { ...renderOptions } = {}) => {
+const render = (
+  ui: React.ReactElement,
+  queryMocks?: readonly MockedResponse<Record<string, any>>[],
+  { ...renderOptions } = {},
+) => {
   const Wrapper: React.FC<{ children?: React.ReactNode }> = ({ children }) => {
     return (
       <ThemeProvider>
```

Comment:
@rot1024 
>On an unrelated topic, I don't like this helper function for unit testing. It's better to directly import testing-library in each test case so that we can see at a glance what context the component requires just by looking at the test code.

The reason why I made this helper function is to test the components which have something to do with `theme` and i18n we need to extend the testing library's `render` function. If we need this helper function, I thought it's more convenient to add gqlMock as well. Is it better to do this on each component test?

Reply:
Basically, I think so. Test code can be thought of as a description of a specification. And the fact that the component depends on some context is also a good specification.

It is not easy to read test code where such specifications cannot be read from the test code or cannot be understood without jumping to the import destination.

Also, when multiple providers are provided in a helper function, it is difficult to know which context the component depends on without reading the component implementation.

Reply:
Thanks. I got the idea. As a refactoring task, we'll fix this part. For now, let it pass this review.